package.json

```diff
@@ -74,7 +74,7 @@
     "ts-jest": "^24.0.0",
     "ts-node": "^8.3.0",
     "tslint": "^5.19.0",
-    "typescript": ">=3.2.1 <3.7.0"
+    "typescript": "beta"
```

Comment:
What's the intent here? `beta` only gets updated, well, when there's a beta (i.e. it won't get any of the changes made in the RC or final release).

Reply:
This was just so that I could get it working, I won't merge it to master like this.

Interestingly, there was a breaking change to the types, which means we can no longer support `<3.7.0` locally (there's a test which relies upon this):

```diff
  interface TypeReference extends ObjectType {
    target: GenericType;
-   typeArguments?: readonly Type[];
+   node?: TypeReferenceNode | ArrayTypeNode | TupleTypeNode;
  }
```
https://github.com/microsoft/TypeScript/pull/33710

So we'll have to make sure the version range is correct when this is merged.